SRC/package.xml

```diff
@@ -2,26 +2,26 @@
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
     <types>
         <members>DreamWorld</members>
-		<members>HelloWorld</members>
-		<members>StudentQuery</members>
+		    <members>HelloWorld</members>
+		    <members>StudentQuery</members>
         <members>Teacher</members>
         <name>ApexClass</name>
     </types>
     <types>
         <members>Dream_World__c.Tesla__c</members>
-		<members>Hello_World__c.Hello__c</members>
+		    <members>Hello_World__c.Hello__c</members>
```

Comment:
Start to use the indentation correctly make it as practice

Reply:
Okay sir